.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+#All Files/PRS
+*                         @pocket/backend
```

Comment:
not sure what this means. Github says this file contains errors

Reply:
Should be fixed now that I've added the team to the repo